examples/activities/resample.js

```diff
@@ -23,21 +23,29 @@ module.exports = function resample(localInputPath, outputUri, options, callback)
       return callback(err);
     }
 
+    // TODO: Shouldn't assume target SRS.
     var args = [
       "-q",
-      "-t_srs", "EPSG:3857",
+//      "-t_srs", "EPSG:3857",
       "-te", options.targetExtent[0], options.targetExtent[1], options.targetExtent[2], options.targetExtent[3],
-      "-tr", options.targetResolution[0], options.targetResolution[1],
-      "-tap",
+//      "-tr", options.targetResolution[0], options.targetResolution[1],
+//      "-tap",
+      "-ts", "256", "256",
```

Comment:
Let's do it later so we can at least iterate on things w/o dealing with versioning yet.